hasher-matcher-actioner/scripts/script_utils.py

```diff
@@ -86,7 +134,137 @@ def send_single_submission_url(
             headers={"content-type": "image/jpeg"},
         )
 
+    def get_content_hash_details(
+        self,
+        content_id: str,
+        api_path: str = "/content/hash/",
+    ):
```

Comment:
typing? Can we re-use the representation classes from hmalib here? I guess that is what you were planning to do. Totally okay to do that as separate PR.

Reply:
all new API methods in this module seem to lack that. You can address my concerns as a reply to this message.

Reply:
I am intentionally avoiding having this class import hmalib so for things like `scripts/storm_submissions_api` we only need this file atm. 

For now scripts that are deeply tied to representation classes from hmalib can import it themselves if/when that becomes impossible I will move them into this class as well. For the time being this file is structured more like `webapp/src/Api.jsx`